package.json

```diff
@@ -49,7 +49,7 @@
     "jest": "^26.6.3",
     "npm-run-all": "^4.1.5",
     "tslint": "^5.6.0",
-    "typescript": "^4.1.3",
+    "typescript": "~4.1.3",
```

Comment:
The commit message says "only minor versions for typescript", I read this to mean only allow updating of minor versions which includes patching. Not a big deal about the actual message but just want to make sure the intention here is to only limit upgrades to patched versions since that is what `~` represents.

Reply:
Yes, my bad with the message, we only want patch

Reply:
fixed